server-script.py

```diff
@@ -0,0 +1,67 @@
+import threading
```

Comment:
Useless script. All logic for handling user connection should be put into class UserHandler

Reply:
Listening you on meeting, I thought it clear what methods class UserHandler should have. But it isn't clear to me right now. UserHandler should have methods like registration, authentication and disconnection?

Reply:
UserHandler class instance should be initialized on every connection to socket and should be able to operate with all types of requests. instance of Userhadler class should be started in a thread in Server